src/main/java/nextstep/subway/line/controller/SectionController.java

```diff
@@ -22,4 +24,11 @@ public ResponseEntity<LineResponseDto> createLine(@PathVariable("id") Long id,
                                                       @RequestBody SectionRequestDto sectionRequestDto) {
         return ResponseEntity.ok().body(lineService.addSection(id, sectionRequestDto));
     }
+
+    @DeleteMapping(value = "/lines/{id}/sections")
+    public ResponseEntity<LineResponseDto> deleteLine(@PathVariable("id") Long id,
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteLine(@PathVariable("id") Long id,
```
아무것도 반환하지 않으니 `Void` 가 적절하지 않을까요?

Reply:
아! 이 부분 놓쳤네요. `LineController`에서 delete 복사 붙여넣기로 넣었는데, 그 부분도 변경해야겠네요 😄 
감사합니다! 🙇